src/bin/lib/subcommands.c/runprogram.h

```diff
@@ -645,7 +645,7 @@ read_into_buf(Program *prog, int filedes, PQExpBuffer buffer, bool error)
 
 		if (prog->processBuffer)
 		{
-			(*prog->processBuffer)(temp_buffer, error);
+			(*prog->processBuffer)(temp_buffer, error, isWarning(temp_buffer));
```

Comment:
The lib/subcommands.c/ files should not be aware of anything that's not in the same directory. That's named a “modularity violation”. Please contain the implementation and call of the `isWarning` function within the src/bin/pgcopydb place. See other comments.